pyquil/tests/test_examples.py

```diff
@@ -0,0 +1,54 @@
+
+import subprocess
+from subprocess import Popen, PIPE, STDOUT
+
+
```

Comment:
prepend all paths with
```python
base_dir = os.path.abspath(os.path.dirname(__file__)+"../../")
```
or ilk?

Reply:
Do we need this?

Reply:
I think as written these tests will only run if you run them from the root directory

Reply:
Well, yeah. Where else would you expect to run the tests from?

Reply:
pycharm (either right clicking on `tests/` to run all the tests or right clicking on `test_examples.py`)

Reply:
It sets the working directory to `pyquil/tests/` presumably

Reply:
OK, I think I've fixed this. Please give pycharm testing a try.